_includes/header.html

```diff
@@ -1,6 +1,15 @@
-{%- assign default_paths = site.pages | map: "path" -%}
-{%- assign page_paths = site.header_pages | default: default_paths -%}
+{%- comment -%}
+This is the page header, and contains the Cylc logo, and menu.
 
+The menu entries are located in the _config.yml file, in the
+project root.
+
+If you want to add a new menu entry, or modify an existing one,
+change that file and either test locally or in your fork.
+
+The checkbox input elements are used for responsive menu, do
+not remove unless you know what you are doing.
+{%- endcomment -%}
```

Comment:
Comments like this are ignored and discarded by Liquid when Jekyll processes these files. Useful only for other developers and contributors.